add matrix (#208)
metalbear-co/docs: docs/reference/release-stages.md

---

> title: Release Staus

Reviewer: Lets move this to the ungrouped section together with third-party and contributing

> title: Release Staus

Reviewer: Noticing some typos ("Staus"), make sure AI goes over PRs before submitting

---

Features in mirrord ship gradually. The release status tells you both how complete the feature is and what level of stability you can expect.

{% hint style="info" %}

> Release staus is currently being mapped and will be added to all feature in the documentation shortly.

Reviewer: Where will it be added to exactly?

> Release staus is currently being mapped and will be added to all feature in the documentation shortly.

Reviewer: ```suggestion
Release status is currently being mapped and will be added to all features in the documentation shortly.
```

{% endhint %}

### `Alpha`

> The feature is available but has a limited scope — not all configurations, platforms, or edge cases are covered yet. You may encounter bugs, unexpected behavior, or performance

Reviewer: ```suggestion
The feature is available but has a limited scope, so not all configurations, platforms, or edge cases are covered yet. You may encounter bugs, unexpected behavior, or performance
```

> issues. APIs and configuration options may change between releases. We actively want feedback at this stage — if something breaks or behaves unexpectedly, please let us know.

Reviewer: ```suggestion
issues. APIs and configuration options may change between releases. We actively want feedback at this stage. If something breaks or behaves unexpectedly, please let us know.
```

> issues. APIs and configuration options may change between releases. We actively want feedback at this stage — if something breaks or behaves unexpectedly, please let us know.

Reviewer: Don't we actively want feedback at all stages?

Author: We do and i written it down also at the bottom.
but the ACTIVELY means we are collecting feedback imo.
I rephrased.
